### ISSUE TITLE
Editing .js file and some editing finger trouble caused Output Message to repeat

### ISSUE THREAD

**Issue by akc42** [Polymer/vscode-plugin#32]
I was editing some (server side) code and got a little bit in a knot with what I was typing and suddenly an output window pops up with lots (15) of these messages in

```
[Error - 08:16:47] Request textDocument/hover failed.
  Message: Request textDocument/hover failed with message: Unexpected token api
  Code: -32603 
```

The code I was editing was the first code line with `api.get` in 

```
      api.get('/emailconf', (req,res, next) => {
      //middleware to check the authentication cookie
      api.use((req,res,next) => {
```

The first line will ultimately become a full function call of api.get before the authentication middleware
I am not 100% sure what kicked this off, but I think I had typed

```
api.get('/emailconf', (req,res, next())
```

And was then back spacing  to remove the brackets at the end of the line

I am not sure what the mouse was doing - I wasn't holding it but may have knocked it over the text with my elbow..

There is a wiggly red line under the last line `api` token.

**Comment by rictic** [comment]
Hm, that's worrying. It sounds like #21 didn't solve the error popup issue

**Comment by rictic** [comment]
Can you confirm that this reproduces in v0.1.2?

**Comment by akc42** [comment]
I picked up v0.1.2 a few minites after you released it, but since you are 8 hours or so behind me I think it must have been late afternoon my time, and I think I reported this prior to that. I can' reproduce this error BUT ...

The one I filed earlier today which is similar is definitely v0.1.2 as it says so in the error message

AND this just happened when I tried to reproduce this problem in the file referenced in the error message that it says already exists.  The output window popped up and this is one copy of a message that repeated twice before I stopped typing.

```
[Error - 18:23:57] Error: Internal error: document server/core/manager.js already exists
    at Analyzer._makeDocument (/home/alan/.vscode/extensions/polymer.polymer-ide-0.1.2/server/node_modules/polymer-analyzer/lib/analyzer.js:147:19)
    at Analyzer.<anonymous> (/home/alan/.vscode/extensions/polymer.polymer-ide-0.1.2/server/node_modules/polymer-analyzer/lib/analyzer.js:132:39)
    at next (<anonymous>)
    at fulfilled (/home/alan/.vscode/extensions/polymer.polymer-ide-0.1.2/server/node_modules/polymer-analyzer/lib/analyzer.js:17:58)
    at process._tickCallback (internal/process/next_tick.js:103:7)
```

**Comment by akc42** [comment]
Actually the error I just reported above is #33 - so its not just an html import

**Comment by rictic** [comment]
I believe this is fixed as of v0.3.1, please open a new issue if you see it again.
